engines/dfc_provider/app/serializers/dfc_provider/catalog_item_serializer.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Serializer used to render a DFC CatalogItem from an OFN Product
+# into JSON-LD format based on DFC ontology
+module DfcProvider
+  class CatalogItemSerializer < ActiveModel::Serializer
+    attribute :id, key: '@id'
+    attribute :type, key: '@type'
+    attribute :references, key: 'dfc:references'
+    attribute :sku, key: 'dfc:sku'
+    attribute :stock_limitation, key: 'dfc:stockLimitation'
+    has_many :offered_through,
+             serializer: DfcProvider::OfferSerializer,
+             key: 'dfc:offeredThrough'
+
+    def id
+      dfc_provider_routes.api_dfc_provider_enterprise_catalog_item_url(
```

Comment:
Some quick tests to check the output of this serializer might be nice, looking at `CatalogItemSerializer#id` and `CatalogItemSerializer#references` in particular.